src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,123 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+Vec3f D_80865900 = { 0.0f, 0.0f, 0.0f };
+
+static void (*D_8086590C[5])(EnPart*, PlayState*) = {
+    func_80865390, func_808654C4, 0x00000000, 0x00000000, 0x00000000,
+};
```

Comment:
- There's no need to use `static` for variables with unique enough names.
- We try to avoid using hardcoded sizes for arrays in general.
- This seems to be an action func, which we prefer to make a typedef for them
- The extra zeroes are probably just file padding and not real.

```suggestion
typedef void (*EnPartActionFunc)(EnPart*, PlayState*);

EnPartActionFunc D_8086590C[] = {
    func_80865390, func_808654C4,
};
```